clang/test/DebugInfo/KeyInstructions/assign-scalar.c

```diff
@@ -0,0 +1,48 @@
+// RUN: %clang_cc1 -gkey-instructions -x c++ %s -debug-info-kind=line-tables-only -emit-llvm -o - \
+// RUN: | FileCheck %s --implicit-check-not atomGroup --implicit-check-not atomRank
```

Comment:
This test is great; I feel we can make it even greater by testing:

  ` g += ++h;`

Or something like that, where there are stores on both sides of a compound assignment expression.